testsuite/shared/src/main/java/org/jboss/as/test/layers/LayersTest.java

```diff
@@ -321,6 +360,27 @@ private static Set<String> appendResult(String title, Result result, StringBuild
         return deltaModules;
     }
 
+    private static String appendExceptionMsg(String existing, String toAppend) {
+        if (!existing.isEmpty()) {
+            existing += '\n';
+        }
+        return existing + toAppend;
+    }
+
+    private static String listModules(Set<String> set, Predicate<String> predicate) {
```

Comment:
To improve the readability, this can be replaced by Collectors.joining(","):

```
return set.stream()
                .filter(s -> {
                    assert s != null;
                    return predicate.test(s);
                })
                .collect(Collectors.joining(","));
```

Reply:
Done